en/docs/design/api-security/oauth2/oauth2-scopes/fine-grained-access-control-with-oauth-scopes.md

```diff
@@ -4,6 +4,14 @@ Scopes enable fine-grained access control to API resources based on user roles.
 
 For example, there can be requirements such as restricting the access to a given API resource to admin users only, while other resources of the same API access should be allowed to consumers with less privileges. Let's see how this kind of role based access control can be managed with the use of OAuth2 scopes.
 
+!!! info
+
+    When creating scopes, it validates the added roles against the underline user store to check if they exist. However, we can override this behavior that it does not validate the roles in the user store. For this purpose, set the Java system property 'disableRoleValidationAtScopeCreation' to 'true' at the server startup:
+         
+         Open <API-M_HOME>/bin/wso2server.(sh|bat) file.
+         Add -DdisableRoleValidationAtScopeCreation=true at the end of the file.
```

Comment:
Instead of above shall we add instructions on both ways,
1. adding it to startup script 
2. providing it during server startup time

similar to https://apim.docs.wso2.com/en/next/install-and-setup/setup/security/logins-and-passwords/set-passwords-using-vars-and-sys-props/#set-passwords-using-system-properties

Reply:
+1